arangodb.go

```diff
@@ -180,5 +179,22 @@ func (a *ArangoDB) UpdateUser(ctx context.Context, req dbplugin.UpdateUserReques
 	a.Lock()
 	defer a.Unlock()
 
-	return dbplugin.UpdateUserResponse{}, errors.New("Not implemented")
+	if req.Password != nil {
```

Comment:
This uses tabs instead of spaces?

Reply:
Well, since I'm using GoLand, from JetBrains, I don't really now.
I'm using the Go formatter that's included, so I'm assuming everything is properly formatted

Reply:
*know

Reply:
I was just asking out of curiosity :+1: